docs/guides/reference/collection_3_summary_grid.md

```diff
@@ -60,22 +64,21 @@ The extent of this grid is based upon the maximal extents of:
 * Digital Earth Australia's Sentinel-2 Collection,
 * With a 3-cell buffer for future growth.
 
-Shown here is an image of the new grid in EPSG:3577 projection, with tiles shown
-where we expect to have land imagery available:
+Shown below is an image of both tile grids and their extents
 
-![](/_files/reference/collection_3_grid.jpeg)
+![](/_files/reference/collection_3_grids.png)
 
-## How can I use this grid?
+## How can I use these grids?
 
-This tiling grid scheme has been used to subdivide processing and
+These tiling grids schema have been used to subdivide processing and
 storage of our continental products.
 
-This tiling grid can be used as an overlay in other spatial systems to
+These tiling grids can be used as an overlay in other spatial systems to
 determine the xMyN reference of a given area of interest from within a
 DEA Collection 3 summary product.
 
-Unfortunately, there is no one-to-one mapping of the new Albers
-Collection 3 grid to the old Albers Collection 2 grid.
+Unfortunately, there is no one-to-one mapping of these Albers
+Collection 3 grid to the older Albers Collection 2 grid.
 
 ## Shift in origin point
 
```

Comment:
This section (e.g. "For a preview..." in particular) likely needs to be revisited too now that no products use the old unshifted grid. We could probably mention that the grid was shifted on date X, but I don't think we should point out to the old unshifted grid any longer - only the up-to-date Landsat and Sentinel-2 grids.